explainer.md

```diff
@@ -245,12 +282,92 @@ and Ubuntu don't support them at all).
+
+On most operating systems, there is only one application specific place to set a badge, rather than one per open window.
+- On Android: The homescreen
+- On OSX: The dock
+- On Windows: The taskbar (users are able to change settings and get a separate icon open window, but this is non-standard).
+- On iOS: The homescreen.
+  
+So it would mean ignoring OS conventions, and it wouldn’t even be possible on most of them!
+
+### Why doesn’t `Badge.set()` apply to the page’s favicon too?
+While in many cases it *is* desirable to have the same app badge and favicon badge (chat applications, email clients and social networks) it doesn’t cater to all use cases.
+
+For example, consider [Github](https://github.com). It has notifications (when someone comments on a pull request, responds to an issue or asks for a review) and statuses (for issues, pull requests and builds).
+
+![Github in a tab, displaying the status of a ready to merge PR and a notification indicator](images/github-tab-status-and-notifications.png)
+
+These two things are conceptually separate: The pull request status belongs to the page while the number of unread notifications corresponds to the conceptual `App` (which is a group of related pages).
+
+![Github in an app window, displaying the status of a ready to merge PR, a notification indicator and a badge](images/github-app-status-and-notifications.png)
+
+Since developers may want to display different "badges" on each surface, we shouldn’t use the same API to set the badge on both surfaces.
+
+An alternative would be to provide an additional API for setting per-tab/window badges, this is discussed [here](https://github.com/WICG/badging/issues/35) and [here](https://github.com/WICG/badging/issues/1).
+
+### What about an overload of the `Badge.set(6, { badgeTabsToo: true })` for badging all favicons? Then sites could opt in to Badging their favicons!
+(this could even be extended with a regex for matching the path of tabs to change the badge on)
+
+It was considered but the overload has a few problems:
+- It’s unclear what the scope of tabs to badge here is (the origin? Pages with the same url? The manifest scope? What if the manifest isn’t linked from this page?).
+- What can be Badged in the OS is [highly constrained by the OS](#specific-operating-system-treatment) (most only support numeric badges, and some don’t even support those) while the badge displayed with favicon hacks can be any image (browser dependent). It would be a shame if the new API was less useful than what developers already have.
+
+If this approach were to be taken, a better solution might be multiple methods inside the Badge namespace:
+
+```js
+Badge.setApp(/* <number> */); // Will be displayed in an OS specific context.
+Badge.setFavIcon(/* <number> | <string> | <image> */); // Can be anything the browser can fit on the favicon.
```

Comment:
As far a "image" goes, the platform already provides a  way of setting the favicon image (and change it dynamically) - so we  probably wouldn't want that at all.